### PR TITLE
Ivan- About section

### PR DESCRIPTION
**Pull request by ivanklchung**
About Section w/o assets

### CODE REVIEW

File: src/app/(landing)/@about/about.module.scss [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    z-index: 1;
+    justify-content: center;
+    align-items: center;
+  }
+  
+  .yellow {
+    color: colors.getColor(warning, 400);
+  }
+
+  .textBox{
```
Comment (fpunny) [inline src/app/(landing)/@about/about.module.scss:21]: Use the flex component

File: src/app/(landing)/@about/about.module.scss [hunk 2]
```diff
@@ -0,0 +1,27 @@
+@use '../../../styles/colors';
+@use '../../../styles/mixins';
+
+.frame {
+    background-color: colors.getColor(secondary, 50);
+  }
+
+  .container {
```
Comment (fpunny) [inline src/app/(landing)/@about/about.module.scss:8]: Use the flex component's props

File: src/app/(landing)/@about/about.module.scss (hunk 1 above)
Comment (fpunny) [inline src/app/(landing)/@about/about.module.scss:21]: Use the Flex component to position instead - The text component just needs an align text

**Review by fpunny — CHANGES_REQUESTED** [review]
Some smol nitpicks that should be addressed - lets make sure we are using the tools to it's fullest

File: src/app/(landing)/@about/page.tsx [hunk 3]
```diff
@@ -1,5 +1,39 @@
...
+import Container from '@/components/Container';
+import Flex from '@/components/Flex';
+import Text from '@/components/Text';
+import styles from './about.module.scss';
+
+function AboutPage() {
+  return (
+    <Container className={styles.frame}>
+      <Flex direction="column" className={styles.container} gap="2x-big">
+        <Text textType="heading-lg" textColor="secondary-900" textWeight="bold">
```
Comment (fpunny) [inline src/app/(landing)/@about/page.tsx:11]: Should have text align for the two texts

File: src/app/(landing)/@about/about.module.scss (hunk 2 above)
Comment (ivanklchung) [reply src/app/(landing)/@about/about.module.scss:8]: got rid of styles.textbox, used Flex component align

File: src/app/(landing)/@about/about.module.scss (hunk 1 above)
Comment (ivanklchung) [reply src/app/(landing)/@about/about.module.scss:21]: done

File: src/app/(landing)/@about/page.tsx (hunk 3 above)
Comment (ivanklchung) [reply src/app/(landing)/@about/page.tsx:11]: I added Flex align component for both

File: src/app/(landing)/@about/about.module.scss [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+    margin-right: auto;
+    position: relative;
+    padding: 12rem 0;
+    z-index: 1;
+    justify-content: center;
+    align-items: center;
```
Comment (fpunny) [inline src/app/(landing)/@about/about.module.scss:14]: Use flexbox props instead

File: src/app/(landing)/@about/page.tsx [hunk 5]
```diff
@@ -1,5 +1,51 @@
...
+          <Text
+            textType="heading-lg"
+            textColor="secondary-900"
+            textWeight="semi-bold"
+            textAlign="center"
+          >
+            for a weekend of memories
+          </Text>
+        </Text>
+        <Flex align="center">
```
Comment (fpunny) [inline src/app/(landing)/@about/page.tsx:32]: Dont think we need this

File: src/app/(landing)/@about/page.tsx [hunk 6]
```diff
@@ -1,5 +1,51 @@
...
+function AboutPage() {
+  return (
+    <Container className={styles.frame}>
+      <Flex
+        align="center"
+        direction="column"
+        className={styles.container}
+        gap="2x-big"
+      >
+        <Text
```
Comment (fpunny) [inline src/app/(landing)/@about/page.tsx:16]: Overall lets use the right html tags for these text elements (h1, h2, p, etc.)

**Review by fpunny — CHANGES_REQUESTED** [review]
Almost there, a few more nit picks and we are good to go

File: src/app/(landing)/@about/page.tsx (hunk 6 above)
Comment (fpunny) [reply src/app/(landing)/@about/page.tsx:16]: @ivanklchung missed one, then we are good

Comment (ivanklchung) [reply src/app/(landing)/@about/page.tsx:16]: okay, i made h2, and p

**Pull request merged**